<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>770</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>770</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>770</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyPix Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>881</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="_tab">
     <attribute name="title">
      <string>Camera</string>
     </attribute>
     <widget class="QPushButton" name="_reset_cumulative">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>510</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset Cumulative</string>
      </property>
     </widget>
     <widget class="QPushButton" name="_rotate_c">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>440</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rotate 90</string>
      </property>
     </widget>
     <widget class="QPushButton" name="_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>680</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="_rotate_cc">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>470</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rotate -90</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="_plainTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>281</width>
        <height>101</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>Welcome to PyPix!
Please connect to camera.</string>
      </property>
     </widget>
     <widget class="PlotWidget" name="_tof_plot" native="true">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>511</width>
        <height>251</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="_progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>281</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Directory</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="_path_button">
         <property name="text">
          <string>Path</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QCheckBox" name="_save_box">
         <property name="text">
          <string>Save Data?</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Filename </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Runtime (s)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="_dir_name"/>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="_file_name">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Iterations</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="_iterations">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="_runtime">
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="value">
          <double>3.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>270</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="_view">
         <item>
          <property name="text">
           <string>Single Shot</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cumulative</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Image Type</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="_colourmap">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Colour Scheme</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_ToT" native="true">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>360</y>
        <width>324</width>
        <height>324</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="_camera_connect_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>281</width>
        <height>36</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="1">
        <widget class="QLineEdit" name="_trim_dir">
         <property name="text">
          <string>FITPix-H08-W0028.xml</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Trim File</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="_trim_button">
         <property name="text">
          <string>Path</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>380</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="1">
        <widget class="QLabel" name="_sensor_temp">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>0.00 C</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Sensor Temp</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Rotation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="_rotation">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_10">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>270</y>
        <width>281</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="1">
        <widget class="QSpinBox" name="_min_y">
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="_min_x">
         <property name="maximum">
          <number>4094</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>ToF Intensity</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="_max_y">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="_max_x">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>500000</number>
         </property>
         <property name="value">
          <number>50000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>ToF Range</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="_tof_range_max">
         <property name="maximum">
          <number>500000</number>
         </property>
         <property name="value">
          <number>50000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="_tof_range_min">
         <property name="maximum">
          <number>4094</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>ToF Selection</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>695</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Scale Image Intensity</string>
      </property>
     </widget>
     <widget class="QSlider" name="_vmax_ToT">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>680</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>99</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>550</y>
        <width>111</width>
        <height>102</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string>Pop-Out Plots!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_pop_tof">
         <property name="text">
          <string>ToF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_pop_toa">
         <property name="text">
          <string>ToA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_pop_tot">
         <property name="text">
          <string>ToT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="_camera_connect">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>320</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Disconnected</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="_ToA" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>360</y>
        <width>324</width>
        <height>324</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="_vmax_ToA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>680</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>99</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>695</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Scale Image Intensity</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_setup"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionMark">
   <property name="text">
    <string>Mark</string>
   </property>
  </action>
  <action name="actionSave_setup">
   <property name="text">
    <string>Save setup</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_trim_dir</tabstop>
  <tabstop>_trim_button</tabstop>
  <tabstop>_path_button</tabstop>
  <tabstop>_save_box</tabstop>
  <tabstop>_plainTextEdit</tabstop>
  <tabstop>_camera_connect_button</tabstop>
  <tabstop>_min_y</tabstop>
  <tabstop>_max_y</tabstop>
  <tabstop>_tof_range_min</tabstop>
  <tabstop>_tof_range_max</tabstop>
  <tabstop>_min_x</tabstop>
  <tabstop>_max_x</tabstop>
  <tabstop>_view</tabstop>
  <tabstop>_colourmap</tabstop>
  <tabstop>_pop_tof</tabstop>
  <tabstop>_pop_toa</tabstop>
  <tabstop>_pop_tot</tabstop>
  <tabstop>_rotate_c</tabstop>
  <tabstop>_rotate_cc</tabstop>
  <tabstop>_reset_cumulative</tabstop>
  <tabstop>_vmax_ToT</tabstop>
  <tabstop>_button</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
